cmd/genericconf/server.go

```diff
@@ -190,19 +190,22 @@ type MetricsServerConfig struct {
 	Addr           string        `koanf:"addr"`
 	Port           int           `koanf:"port"`
 	Pprof          bool          `koanf:"pprof"`
+	PprofPort      int           `koanf:"pprof-port"`
 	UpdateInterval time.Duration `koanf:"update-interval"`
 }
 
 var MetricsServerConfigDefault = MetricsServerConfig{
 	Addr:           "127.0.0.1",
 	Port:           6070,
-	Pprof:          false,
+	Pprof:          true,
+	PprofPort:      6071,
 	UpdateInterval: 3 * time.Second,
 }
 
 func MetricsServerAddOptions(prefix string, f *flag.FlagSet) {
 	f.String(prefix+".addr", MetricsServerConfigDefault.Addr, "metrics server address")
 	f.Int(prefix+".port", MetricsServerConfigDefault.Port, "metrics server port")
 	f.Bool(prefix+".pprof", MetricsServerConfigDefault.Pprof, "enable profiling for Go")
```

Comment:
I'm not sure which of these we should do, but we should either:
- Add a comment here explaining that pprof will only be exposed if metrics are enabled
- Make pprof independent of metrics, and have it default to disabled.

I think the latter approach is a bit better because metrics and pprof seem like separate features to me.

Reply:
Went with the second option.